spec/rubocop/cop/rails/unknown_env_spec.rb

```diff
@@ -72,4 +83,30 @@
       Rails.env == 'production'
     RUBY
   end
+
+  context 'Rails 7.1' do
+    let(:config) do
+      RuboCop::Config.new(
+        {
+          'AllCops' => {
+            'TargetRailsVersion' => '7.1'
+          },
+          'Rails/UnknownEnv' => {
+            'Environments' => %w[
+              development
+              production
+              test
+            ]
+          }
+        }
+      )
+    end
+
+    it 'accepts local as an environment name on Rails 7.1' do
```

Comment:
Now that #1134 has been merged, I think this could be changed to use the shared context and `cop_config` defined above. We also don't need to repeat "Rails 7.1" in the spec name, as it is already in the context name.


```suggestion
  context 'in Rails 7.1 onwards', :rails71 do
    it 'accepts local as an environment name' do
```

Reply:
@ghiculescu ping.